docs/layers/geojson-layer.md

```diff
@@ -7,70 +7,33 @@
 The GeoJson Layer takes in [GeoJson](http://geojson.org/) formatted data and
 renders it as interactive polygons, lines and points.
 
-  <div align="center">
-    <img height="300" src="/demo/src/static/images/demo-thumb-geojson.jpg" />
-  </div>
-
-```
-import {GeoJsonLayer} from 'deck.gl';
-
-new GeoJsonLayer({
-  data: {
-    "type": "FeatureCollection",
-      "features": [
-        {
-          "type": "Feature",
-          "geometry": {
-            "type": "Point",
-            "coordinates": [102.0, 0.5]
-          },
-          "properties": {
-            "color": [255, 0, 0, 255],
-            "prop1": {"application": "defined"}
-          }
-        },
-        ...
-      ]
-    }
-  }
-});
-```
-
-Remarks:
-* By supplying a GeoJson `FeatureCollection` you can render multiple polygons,
-  lines and points.
-* Each Feature has an optional "properties" object. The layer will look
-  for an optional property `color`, which is expected to be a 4 element
-  array of values between 0 and 255, representing the rgba values for
-  the color of that `Feature`.
+<div align="center">
+  <img height="300" src="/demo/src/static/images/demo-thumb-geojson.jpg" />
+</div>
 
+    import {GeoJsonLayer} from 'deck.gl';
 
 ## Properties
 
 Inherits from all [Base Layer properties](/docs/layers/base-layer.md),
 however, the `data` prop is interpreted slightly more flexibly to accommodate
-puree GeoJson "payloads".
+pure GeoJson "payloads".
 
-Also note that the `GeoJsonLayer` is a **composite** layer, that renders points
-using `ScatterplotLayer`, lines and polygon outlines (`stroked` polygons)
-using `PathLayer`, and `filled` polygons using a primitive polygon layer.
-The `GeoJsonLayer` passes through most of its props to these underlying layers
-so you can sometimes achieve additional control of rendering by studying
-the props of the generated layers.
+Also note that the `GeoJsonLayer` is a **composite** layer, that renders points, lines and polygons with its outlines with different "underlying" layers, so developers can sometimes achieve additional control of rendering by controlling
+the props of the underlying layers directly.
 
 #### `data` (Object, Array or iterable)
 
 A GeoJson `FeatureCollection` object, or a list of GeoJson `Feature` objects.
 For more information on how the GeoJson is interpreted see the introduction.
 
-
 #### `filled` (Boolean, optional)
 
 - Default: `true`
 
 Whether to draw filled polygons (solid fill). Note that for each polygon,
-only the area between the outer polygon and any holes will be filled.
-
+only the area between the outer polygon and any holes will be filled. This
+prop is effective only when the polygon is NOT extruded.
```

Comment:
Aren't extruded polygons always filled unless wireframe is true? While technically correct, the sentence could be read to say they are not.